packages/widget/src/controllers/transfers/fungible-token-transfer.ts

```diff
@@ -219,8 +321,9 @@ export class FungibleTokenTransferController implements ReactiveController {
     if (this.resourceAmount.eq(0)) {
       return FungibleTransferState.MISSING_RESOURCE_AMOUNT;
     }
-
-    // Enabled States
+    if (this.destinationAddress === '') {
+      return FungibleTransferState.MISSING_DESTINATION_ADDRESS;
+    }
     if (
```

Comment:
it's a duplication, we already have this code above

Reply:
removed